src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,11 +5,27 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
```

Comment:
Don't begin class or method implementation with an empty line. It is unnecessary and does not follow Java code style conventions.